package.json

```
@@ -43,8 +43,8 @@
     "mock-test": "./script/mock-test.bash",
     "test-no-es6-browser-globals": "node dist/es6/clipboard-polyfill.es6.js",
     "dev": "node script/dev.js",
-    "lint": "npx rome check ./script ./src",
-    "format": "npx rome format --write ./script ./src",
+    "lint": "npx biome check ./script ./src",
+    "format": "npx biome format --write ./script ./src",
```

Nit: I prefer to use fully qualified package names.
```suggestion
    "lint": "npx @biomejs/biome check ./script ./src",
    "format": "npx @biomejs/biome format --write ./script ./src",
```